packages/webpack/test/fixtures/main/simple.js

```diff
@@ -0,0 +1,3 @@
+// this throws if we scuttled props of compartment globalThis instead of startCompartment globalThis by mistake
```

Comment:
(suggestion) I like Endo's convention of adding a `README` for each fixture

Reply:
can i see an example?

Reply:
see `endojs/endo/packages/compartment-mapper/test/fixture/*/node_modules/*/README.md` :D

Reply:
i think it'll be strange if `simple` ends up being the only file there with a README

Reply:
not sure what to do exactly, because it's not fully comparable with Endo - do we want fixtures (main, hack1, hack2) to become folders? as in:

present:
main.js

future:
/main/
  index.js
  README.md

?

Reply:
https://github.com/LavaMoat/LavaMoat/pull/1517